scss/standalone/patterns_grid-breakout-full.scss

```diff
@@ -0,0 +1,6 @@
+@import '../base';
+@include vf-base;
+
+@import '../patterns_grid';
+@include vf-p-grid;
+@include vf-l-fluid-breakout(13rem, 100%, 0, '--single-main');
```

Comment:
You need to `@import '../layouts_fluid-breakout';` because it's moved to separate file now.

Reply:
done